tas-installer/internal/kubernetes/namespace.go

```diff
@@ -0,0 +1,51 @@
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (kc *KubernetesClient) CreateNamespace(ns string) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	exists, err := kc.namespaceExists(ctx, ns)
+	if err != nil {
+		return err
+	}
+
+	if !exists {
+		namespace := &v1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: ns,
+			},
+		}
+
+		_, err := kc.Clientset.CoreV1().Namespaces().Create(ctx, namespace, metav1.CreateOptions{})
+		if err != nil {
+			return err
+		}
+		fmt.Printf("%s namespace created successfully \n", ns)
+	} else {
+		fmt.Printf("%s namespace already exists, skipping create \n", ns)
```

Comment:
It's not a huge deal, but I've mentioned it elsewhere, so why not be consistent. My opinion is that it's best to keep user facing stuff in the `cmd` package. If you want to print out, success/failure messages, check the return status on `kc.createNamespace(...)` in installer.go. It does make installer.go a little less "clean" looking. But it does work nicely for separation of concerns.